README.md

```diff
@@ -115,6 +115,7 @@ For syntax, see [lookup.json](./lib/lookup.json), the available attributes are:
 "repo": "https://github.com/pugjs/jade" - Use a different github repo
 "skipAnsi": true             Strip ansi data from output stream of npm
 "script": /path/to/script | https://url/to/script - Use a custom test script
+"scriptWin": \path\to\script | https://url/to/script - Use a custom windows test script
```

Comment:
I guess if `script` can be a string or an object that's semver-minor? I agree allowing it to be an object gives a lot more freedom.

Could we maybe use the same options we use for skipping tests? That way you could pass different scripts for different versions of node or whatever (and the syntax would match, consistency is always nice).